src/components/runtime-common.js

```diff
@@ -145,6 +147,7 @@ export const SaveFilesHelpGalaxy = () => {
     ]),
     p(['If you want to save some files permanently, such as input data, analysis outputs, or installed packages, ',
       h(Link, {
+        'aria-label': 'Galaxy save help',
```

Comment:
Since this will be read as a sentence, 'move them to the workspace bucket' makes more sense to me than this label. If anything, we might insert the text 'learn how to' before the link, or something similar.
```suggestion
```